modules/dhcp/dhcp_api.rb

```diff
@@ -1,3 +1,7 @@
+require "proxy/validations"
+
+include Proxy::Validations
```

Comment:
This smells, either include this into a class or just avoid including and use full namespace.